docs/source/configuration_file.md

```diff
@@ -0,0 +1,139 @@
+# Configuration file
```

Comment:
i would definitely reference this file in the `index.rst`, between getting_started.md and tutorials.md.

Reply:
Done :-)